domain/schema/changelog.go

```diff
@@ -4,11 +4,11 @@
 package schema
 
 import (
-	"fmt"
-
 	"github.com/juju/juju/core/database/schema"
 )
 
+type tableNamespaceID = int
```

Comment:
I think we should rethink this at some point. It doesn't provide strong guarantees between schema and change log watchers.